modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java

```diff
@@ -515,6 +532,10 @@ else if (Objects.equals(objectFieldDBType, "Long")) {
 	private static final Log _log = LogFactoryUtil.getLog(
 		ObjectDDMStorageAdapter.class);
 
+	private final Set<String> _ddmFormfieldTypesNotMappable = SetUtil.fromArray(
```

Comment:
I remembered that these field types are categorized as transient in the DDMForm context. Transient fields are fields that don't save values from the user. 
Saying that, we could rename this variable.